src/igvfd/tests/test_schema_whole_organism.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+
+def test_accession(whole_organism1, testapp):
+    res = testapp.get(whole_organism1['@id'])
+    assert res.json['accession'][:6] == 'IGVFSM'
+
+
+def test_lot_id_dependency(whole_organism1, testapp):
+    res = testapp.patch_json(
+        whole_organism1['@id'],
+        {'lot_id': 'R00002'}, expect_errors=True)
+    assert res.status_code == 422
+
+
+def test_description(whole_organism1, testapp):
+    res = testapp.patch_json(
+        whole_organism1['@id'],
+        {'description': 'I am a modern major general.'})
+    assert res.status_code == 200
+
+
+def test_failure_patch_calculated_sex(whole_organism1, testapp):
+    res = testapp.patch_json(
+        whole_organism1['@id'],
+        {'sex': 'female'}, expect_errors=True)
```

Comment:
you are testing that the sex is calculated, but haven't removed it from the schema?

Reply:
I know. My bad!